more/free-programming-cheatsheets.md

```diff
@@ -125,6 +126,11 @@
 * [Dart Quick Reference](https://koenig-media.raywenderlich.com/uploads/2019/08/RW-Dart-Cheatsheet-1.0.2.pdf) - Koenig-media.raywenderlich.com (PDF)
 
 
+### DBMS
+
+* [DBMS_Cheatsheet](https://annamalaiuniversity.ac.in/studport/download/engg/CSE_Engg/resources/CSE_CSPC403_DBMS.pdf) - Annamalai University DBMS Course Cheatsheet (PDF)
```

Comment:
I don't think I would consider a 100 page PDF a cheat sheet. I think this would fit better books or courses.

Reply:
Okay, will add it in books.